feature-detects/css/animations.js

```diff
@@ -8,7 +8,7 @@
   "warnings": ["Android < 4 will pass this test, but can only animate a single property at a time"],
   "notes": [{
     "name" : "Article: 'Dispelling the Android CSS animation myths'",
-    "href": "http://goo.gl/OGw5Gm"
+    "href": "https://daneden.me/2011/12/14/putting-up-with-androids-bullshit/"
```

Comment:
We actually short url'd this because the use of bullshit didn't pass some check in visual studio believe it or not https://plus.google.com/u/0/+shanselman/posts/Kfq6eWrueCz?cfem=1